libgloss/arc/emsk.specs

```diff
@@ -3,7 +3,7 @@
 %rename startfile emsk_startfile
 
 *link:
-%(emsk_link) -marcv2elfx
```

Comment:
Please see above, probably you need to make for each emsk variant a spec file containing the correct memory values.